### DISCUSSION TITLE
2.0.7

### DISCUSSION THREAD

**Discussion by benhillis** [microsoft/WSL#10694] [General]
*  Only mirror supported proxy settings when autoproxy is enabled
*  Plumb loopback routes during both IP address add and IP address update operations
*  Add more actionable error message for when Virtual Machine Platform is not present (solves #10666)
*  Add error handling to support servercore SKU

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WSL/releases/tag/2.0.7'>2.0.7</a>.</em>

**Comment by andersonkyle** [comment]
@benhillis Is there any way to install a specific version of WSL?  

I recently installed `2.0.6` and then `2.0.7` via `wsl --update --pre-release` to test out some of the new features but am running into a few issues and was curious if there was an easy way to go back to a specific version? (e.g. `wsl --install-wsl 1.2.5`)

This would also be useful in trying to identify which version of WSL introduced a bug.

**Reply by EastMonster** [reply]
You can download `.msi` installer of your desired version from the release page.

**Reply by jcrben** [reply]
@andersonkyle you can write a script which installs it via the add-appx command:
`powershell -command "add-appxpackage -Path <path-to-msixbundle>"`
